include/LightGBM/c_api.h

```diff
@@ -758,25 +825,26 @@ LIGHTGBM_C_EXPORT int LGBM_BoosterPredictForCSC(BoosterHandle handle,
                                                 double* out_result);
 
 /*!
-* \brief make prediction for an new data set
-*        Note:  should pre-allocate memory for out_result,
-*               for normal and raw score: its length is equal to num_class * num_data
-*               for leaf index, its length is equal to num_class * num_data * num_iteration
-* \param handle handle
-* \param data pointer to the data space
-* \param data_type type of data pointer, can be C_API_DTYPE_FLOAT32 or C_API_DTYPE_FLOAT64
-* \param nrow number of rows
-* \param ncol number columns
-* \param is_row_major 1 for row major, 0 for column major
-* \param predict_type
-*          C_API_PREDICT_NORMAL: normal prediction, with transform (if needed)
-*          C_API_PREDICT_RAW_SCORE: raw score
-*          C_API_PREDICT_LEAF_INDEX: leaf index
-* \param num_iteration number of iteration for prediction, <= 0 means no limit
-* \param parameter Other parameters for the parameters, e.g. early stopping for prediction.
-* \param out_len len of output result
-* \param out_result used to set a pointer to array, should allocate memory before call this function
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_BoosterPredictForMat
```

Comment:
The same comments as for `LGBM_BoosterPredictForCSR`, if other ones are not presented.